streamly.cabal

```diff
@@ -289,9 +289,11 @@ library
                        Streamly.Memory.Malloc
                      , Streamly.Memory.Ring
 
-                     , Streamly.FileSystem.IOVec
-                     , Streamly.FileSystem.FDIO
-                     , Streamly.FileSystem.FD
+    if impl(ghc < 9)
+      other-modules:
+          Streamly.FileSystem.IOVec
+        , Streamly.FileSystem.FDIO
+        , Streamly.FileSystem.FD
```

Comment:
Are these not supported post ghc 9?

Reply:
This is not a new change. It is included in #2421.